.github/workflows/build_monolithic.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     env:
       # crossover source code to be downloaded
       CROSS_OVER_SOURCE_URL: https://media.codeweavers.com/pub/crossover/source/crossover-sources-21.2.0.tar.gz
```

Comment:
Should this be updated too?

Reply:
Technically this doesn't need to be updated as CodeWeavers are no longer updating there bundled copy of DXVK but I guess there shouldn't be any harm in doing so.